.gitignore

```diff
@@ -14,3 +14,6 @@ build/
 
 # Coverage reports
 .jacoco/
+
+#Ignore all .deb files
+*.deb

```

Comment:
@Xenowa Why we had to put this entry? if its created inside build directory, can't we just ignore that?

Reply:
This entry is needed to address an issue where the GitHub package publishing fails. The failure occurs because the setup Ballerina action downloads a `.deb` file, which is then flagged as uncommitted changes. By explicitly ignoring this `.deb` file we can ensure that this file does not interfere with the publishing process.

Reply:
Can you create an issue in [setup-ballerina](https://github.com/ballerina-platform/setup-ballerina/) repo? Ideally, it should clean everything after setting up and should not keep anything to be committed.

Reply:
We too faced this when we did the connector revamp. We fixed it by adding it to the .gitignore file. Would be great if we can fix this from the setup-ballerina action itself. 
@NipunaRanasinghe FYI.

Reply:
@ThisaruGuruge Yes I agree. Otherwise we'll have keep doing this extra step in all the future connectors as well. +1 to fix this behaviour from the action side itself.

Reply:
Yes we need to fix it there. Once we fix that we should be able to remove these everwhere